api/http/proxy/factory/docker/transport.go

```diff
@@ -492,6 +492,39 @@ func (transport *Transport) restrictedResourceOperation(request *http.Request, r
 			return utils.WriteAccessDeniedResponse()
 		}
 	}
+	accessContext, err := transport.createRegistryAccessContext(request)
```

Comment:
this code already exists in `replaceRegistryAuthenticationHeader`, maybe try to use it?

Reply:
It's updated:)